app/assets/src/components/common/CoverageVizBottomSidebar/HitGroupViz.jsx

```diff
@@ -209,21 +212,44 @@ export default class HitGroupViz extends React.Component {
           trigger={
             <div
               className={cx(cs.icon, cs.downloadIcon)}
-              onClick={this.handleContigDownload}
+              onClick={withAnalytics(
+                this.handleContigDownload,
+                "HitGroupViz_contig-download-button_clicked",
+                {
+                  contigByteranges: get(
```

Comment:
Not sure what `contigByteranges` is, but I advise not clogging up our analytics tables with complex data structures. Use `contigByteranges.length` or try to break it up into several scalar values.

Reply:
Done!